### PR TITLE
DOCS-9778 expand RTF docs for built-in alerts

### PR DESCRIPTION
**Pull request by IsaacEldridge**
Hey @sarjeet2013, this is a ticket I pulled from the RTF docs backlog I inherited, originally based on https://www.mulesoft.org/jira/browse/SE-18892. Is it possible to get someone to help with the descriptions for these alerts? Or if you point me in the right direction, I can try to self-serve. 

cc @kmendiratta-crm

### CODE REVIEW

File: modules/ROOT/pages/configure-alerting.adoc [hunk 1]
```diff
@@ -72,11 +72,22 @@ The following table lists some of default alerts provided by Anypoint Runtime Fa
...
+| `etcd_latency_batch` | 
+| `filesystem` | Sends a warning when > 80% used. Triggers a critical error when > 90% used.
+| `heap_threshold` |
+| `high-cpu` | Sends a warning when > 75% used. Triggers a critical error when > 90% used.
+| `high_memory` | Sends a warning when > 80% used. Triggers a critical error when > 90% used.
+| `influxdb_health_batch` |
+| `ingress` |
+| `kubernetes_node` | Sends a warning when > 90% used. Triggers a critical error when > 95% used.
+| `muleapps` |
+| `network_connectivity` |
```
Comment (kmendiratta-crm) [inline modules/ROOT/pages/configure-alerting.adoc]: The name of this alert is `net_connectivity` to be consistent

Comment (IsaacEldridge) [inline modules/ROOT/pages/configure-alerting.adoc]:
```suggestion
| `net_connectivity` |
```

**Review by kmendiratta-crm — DISMISSED** [review]

File: modules/ROOT/pages/configure-alerting.adoc [hunk 2]
```diff
@@ -72,11 +72,20 @@ The following table lists some of default alerts provided by Anypoint Runtime Fa
 .Runtime Fabric Default Alerts
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/configure-alerting.adoc]:
```suggestion
```

File: modules/ROOT/pages/configure-alerting.adoc [hunk 3]
```diff
@@ -69,14 +69,22 @@ Currently, only one alerts email recipient is supported.
...
+| `filesystem` | Sends a warning when > 80% used. Triggers a critical error when > 90% used.
+| `high-cpu` | Sends a warning when > 75% used. Triggers a critical error when > 90% used.
+| `high_memory` | Sends a warning when > 80% used. Triggers a critical error when > 90% used.
+| `influxdb_health_batch` | Triggers an error if InfluxDB is down or if there is no connection between Kapacitor and InfluxDB.
+| `ingress` | Triggers a critical error when > 80% used.
+| `kubernetes_node` | Sends a warning when > 90% used. Triggers a critical error when > 95% used.
+| `muleapps` | Triggers a critical error when > 90% used. 
+| `network_params` | Triggers a critical error if networking is disabled on a node. 
+| `uptime` | Sends a warning when the system is unaccessible for longer than 5 min.
+
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/configure-alerting.adoc]:
```suggestion
```

File: modules/ROOT/pages/configure-alerting.adoc [hunk 4]
```diff
@@ -69,14 +69,22 @@ Currently, only one alerts email recipient is supported.
...
+
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/configure-alerting.adoc]:
```suggestion
```

File: modules/ROOT/pages/configure-alerting.adoc [hunk 5]
```diff
@@ -66,20 +66,25 @@ Currently, only one alerts email recipient is supported.
...
 [%header,cols="2*a"]
-.Runtime Fabric Default Alerts
 |===
 |Alert | Description
-| High CPU usage | Sends a warning when > 75% used. Triggers a critical error when > 90% used.
-| High memory usage | Sends a warning when > 80% used. Triggers a critical error when > 90% used.
-| High disk space usage | Sends a warning when > 80% used. Triggers a critical error when > 90% used.
-| High node usage | Sends a warning when > 90% used. Triggers a critical error when > 95% used.
-| etcd instance health | Sends an error when the etcd master is unaccessible for longer than 5 min.
+| `docker` | Sends an error when docker daemon is down.
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/configure-alerting.adoc]:
```suggestion
| `docker` | Sends an error when the Docker daemon is down.
```

File: modules/ROOT/pages/configure-alerting.adoc [hunk 6]
```diff
@@ -66,20 +66,25 @@ Currently, only one alerts email recipient is supported.
...
+| `etcd_latency_batch` | Sends a warning when follower-leader latency exceeds 500ms, and triggers an error if it exceeds 1 second over a one minute period.
+| `filesystem` | Sends a warning when 80% of capacity is used. Triggers a critical error when more than 90% is used.
+| `high-cpu` | Sends a warning when 75% of capacity used. Triggers a critical error when more than 90% is used.
+| `high_memory` | Sends a warning when 80% of memory is used. Triggers a critical error when more than 90% is used.
+| `influxdb_health_batch` | Triggers an error if InfluxDB is down or if there is no connection between Kapacitor and InfluxDB.
+| `ingress` | Triggers a critical error when more than 80% of capacity is used.
+| `kubernetes_node` | Sends a warning when more than 90% of CPU is used. Triggers a critical error when more than 95% is used.
+| `muleapps` | Triggers a critical error when more than 90% of CPU is used. 
+| `network_params` | Triggers a critical error if networking is disabled on a node. 
+| `uptime` | Sends a warning when the system is unaccessible for longer than 5 min.
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/configure-alerting.adoc]:
```suggestion
| `uptime` | Sends a warning when the system is inaccessible for longer than 5 min.
```

File: modules/ROOT/pages/configure-alerting.adoc [hunk 7]
```diff
@@ -66,20 +66,25 @@ Currently, only one alerts email recipient is supported.
...
 |===
 |Alert | Description
-| High CPU usage | Sends a warning when > 75% used. Triggers a critical error when > 90% used.
-| High memory usage | Sends a warning when > 80% used. Triggers a critical error when > 90% used.
-| High disk space usage | Sends a warning when > 80% used. Triggers a critical error when > 90% used.
-| High node usage | Sends a warning when > 90% used. Triggers a critical error when > 95% used.
-| etcd instance health | Sends an error when the etcd master is unaccessible for longer than 5 min.
+| `docker` | Sends an error when docker daemon is down.
+| `etcd` | Sends an error when the etcd master is unaccessible for longer than 5 min.
+| `etcd_latency_batch` | Sends a warning when follower-leader latency exceeds 500ms, and triggers an error if it exceeds 1 second over a one minute period.
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/configure-alerting.adoc]:
```suggestion
| `etcd_latency_batch` | Sends a warning when follower-leader latency exceeds 500ms, and triggers an error if it exceeds 1 second over a one-minute period.
```

**Review by valkyrie69 — APPROVED** [review]

**Pull request merged**